src/prefect/server/api/server.py

```diff
@@ -774,7 +774,12 @@ def start(self, timeout: Optional[int] = None):
             if self.port is None:
                 self.port = self.find_available_port()
             assert self.port is not None, "Port must be provided or available"
-            subprocess_server_logger.info(f"Starting server on {self.address}")
+            help_message = (
+                f"Starting ephemeral server on {self.address} - see "
```

Comment:
two thoughts:
- maybe call it a "temporary" server for the purposes of this message
- suggest adding a newline: 
```suggestion
                f"Starting ephemeral server on {self.address}\nSee "
```

Reply:
makes sense - updated!